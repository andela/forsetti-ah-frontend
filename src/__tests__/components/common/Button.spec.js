import React from 'react';
import renderer from 'react-test-renderer';
import ButtonComponent from '../../../components/common/Button';

describe('<ButtonComponent  />', () => {
  it('renders the ButtonComponent  correctly', () => {
    const tree = renderer.create(<ButtonComponent />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
