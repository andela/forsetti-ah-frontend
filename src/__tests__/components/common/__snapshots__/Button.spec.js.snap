// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> renders correctly 1`] = `
<button
  aria-label={null}
  className="btn btn-secondary"
  onClick={[Function]}
/>
`;
